packages/core/content-releases/server/src/bootstrap.ts

```diff
@@ -103,46 +54,11 @@ export const bootstrap = async ({ strapi }: { strapi: Core.Strapi }) => {
           });
```

Comment:
this lifecycle is blocking , is it necessary? Maybe we can make this asynchronous so it doesnt slow down deleeteMany ops